pkgs/applications/misc/chrysalis/default.nix

```diff
@@ -2,34 +2,40 @@
 
 let
   pname = "chrysalis";
-  version = "0.12.0";
-in appimageTools.wrapAppImage rec {
+  version = "0.13.2";
   name = "${pname}-${version}-binary";
-
-  src = appimageTools.extract {
-    inherit name;
-    src = fetchurl {
-      url = "https://github.com/keyboardio/${pname}/releases/download/v${version}/${pname}-${version}.AppImage";
-      sha256 = "sha256-sQoEO1UII4Gbp7UbHCCyejsd94lkBbi93TH325EamFc=";
-    };
+  src = fetchurl {
+    url =
+      "https://github.com/keyboardio/${pname}/releases/download/v${version}/${pname}-${version}-x64.AppImage";
+    hash =
+      "sha512-WuItdQ/hDxbZZ3zulHI74NUkuYfesV/31rA1gPakCFgX2hpPrmKzwUez2vqt4N5qrGyphrR0bcelUatGZhOn5A==";
   };
+  appimageContents = appimageTools.extract { inherit name src; };
+in appimageTools.wrapType2 rec {
+  inherit name pname src;
```

Comment:
I'm not unclear which values do and do not need to be explicitly inherited. Any guidance would be appreciated...

Reply:
https://github.com/nixos/nixpkgs/blob/master/pkgs/build-support/appimage/default.nix#L60-L72
If you didn't specify `name`, you would need to specify `pname` and `version`. Since `name` is specified it seems fine.

Reply:
Should I leave this as is or make changes?